assets/src/design-system/theme/typography.js

```diff
@@ -27,7 +27,7 @@ const themeFonts = {
 export const typography = {
   family: { ...themeFonts },
   weight: {
-    bold: 500,
```

Comment:
This got reverted back to 500 somewhere with our many PRs 😅  the good news is that i think nothing else until now was really impacted by this, bold is getting handled by typography presets.